.github/workflows/channels-ci.yml

```diff
@@ -151,13 +151,6 @@ jobs:
           npm run test-ci --workspace=channels
           npm run test-ci --workspace=platform/client
           npm run test-ci --workspace=playbooks
-      - name: ci/publish-test-results
-        uses: EnricoMi/publish-unit-test-result-action@a3caf02865c0604ad3dc1ecfcc5cdec9c41b7936  # v2.3.0
-        if: always()
-        with:
-          junit_files: "webapp/channels/build/**/*.xml"
-          comment_mode: always
```

Comment:
I suggest you change this to failure https://github.com/EnricoMi/publish-unit-test-result-action#configuration